api/routes.py

```diff
@@ -677,13 +677,13 @@ def track_analytics_event(identifier_type, identifier, event_type):
     "/playtimes/<int:collection_id>/<identifier_type>/<path:identifier>",
     methods=["POST"],
 )
+@has_library
+@requires_auth
```

Comment:
We previously had the `allow_auth` decorator, which did not require authentication. I reordered the decorators so that authentication would happen first. Before I reordered them, Flask-Pydantic would validate the payload first and would return a response for validation errors, even when no authentication was provided.